### PR TITLE
Added sprint 1 work and tests

### PR DESCRIPTION
**Pull request by Jarbo52**
Added my two users stories and two tests cases for each and extra test cases for helper methods

### CODE REVIEW

File: examples.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+examplePersonAlive = {
```
Comment (DQSevilla) [inline examples.py:1]: Imagine using camelCase in python which universally is written in snake_case

File: project03.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
 from prettytable import PrettyTable
 from datetime import date
+import datetime
+import time
 
-GEDCOM_FILE = 'cs555project01.ged'
+GEDCOM_FILE = 'cs555project03.ged'
```
Comment (DQSevilla) [inline project03.py:6]: We need files as input, I'll handle that during this sprint

File: unittests.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+class TestMarriageMethods(unittest.TestCase):
+
+    def testTimestamp(self):
+        self.assertEqual(project03.gedcomDateToUnixTimestamp('01 JAN 1970'), time.mktime(datetime.datetime.strptime('1/1/1970', '%d/%m/%Y').timetuple()))
+        self.assertTrue(project03.gedcomDateToUnixTimestamp('28 SEP 2020') == project03.gedcomDateToUnixTimestamp('28 SEP 2020'))
+        self.assertTrue(project03.gedcomDateToUnixTimestamp('28 SEP 2020') > project03.gedcomDateToUnixTimestamp('28 SEP 2019'))
+
+    def testMarriageBeforeDivorceSuccess(self):
+        self.assertTrue(project03.verifyMarriageBeforeDivorce(examples.exampleFamilyDivorced))
```
Comment (DQSevilla) [inline unittests.py:15]: If we update the way our dicts work internally we also have to update how examples work which sucks. We should have a blackbox (tested) function that parses gedcom into the dict and have one or more example gedcom files that we can parse into dicts and then pass in. Refactor work for next sprint?

**Comment by DQSevilla** [comment]
Your commits are still not signed btw

**Review by DQSevilla — APPROVED** [review]

File: examples.py (hunk 1 above)
Comment (Jarbo52) [reply examples.py:1]: Welp

File: project03.py (hunk 2 above)
Comment (Jarbo52) [reply project03.py:6]: Yeah he never specified how he wants input

File: unittests.py (hunk 3 above)
Comment (Jarbo52) [reply unittests.py:15]: Sounds good

**Pull request merged**